spk/mosh/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = mosh
+SPK_VERS = 1.3.2
+SPK_REV = 1
+DSM_UI_DIR = app
```

Comment:
This line is useless as you provide no `app` folder. It explains why you got `convert` failures at build... even with my optional `SPK_ICON` support. Please remove it.